backend/app/notify/notify.go

```diff
@@ -29,14 +29,27 @@ type Destination interface {
 	Send(ctx context.Context, req request) error
 }
 
-// Store defines the minimal interface accessing stored comments used by notifier
+// VerificationDestination defines interface for a given destination service which supports verification sending
```

Comment:
I don't think it should be part of notify at all. To me Verification is rather concern from another world, probably from service.Email